projects/BFDR/IJEBirth.cs

```diff
@@ -1767,7 +1767,7 @@ public string FRACE22
             {
                 if (!String.IsNullOrWhiteSpace(value))
                 {
-                    Set_FatherRace(NvssRace.FirstOtherRaceLiteral, value);
+                    Set_FatherRace(NvssRace.FirstOtherRaceLiteral, value.Trim());
```

Comment:
same here, should the Trim be moved into Set_FatherRace to make sure it's always trimmed?